SPECS/moby-engine/enable-docker-proxy-libexec-search.patch

```diff
@@ -0,0 +1,86 @@
+From f8c088be055b72e58005ef9e56cf4f4008bbc5dd Mon Sep 17 00:00:00 2001
```

Comment:
Wondering if we should cherry-pick this to `3.0-dev` as well ?

Reply:
Yeah, going to get it into 3.0-dev as well. Focused on getting approval for the 2.0 change first in case it needs any modifications.